src/main/kotlin/no/nav/omsorgspenger/sak/HentIdentPdlMediator.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.omsorgspenger.sak
+
+import kotlinx.coroutines.runBlocking
+import no.nav.omsorgspenger.client.pdl.PdlClient
+
+internal class HentIdentPdlMediator(
+        internal val pdlClient: PdlClient
+) {
+
+    fun hentIdenter(identer: Set<String>): Map<String, Set<String>> {
```

Comment:
Lage et alias som heter `typealias Identitetsnummer = String` så blir dette litt enklere å lese.

Også kan jo vi kalle denne funksjonen for hentIdentitetsnumer ettersom vi ikke henter noen andre type identer